### PR TITLE
Add new keyboard Plaid and ATMEGA328p support

### PR DESCRIPTION
**Pull request by hsgw**
## Description
- Add new keyboard Plaid.
- Add atmega328p pin/port defines to quantum/config_common.h for that.
- Trivial fix in tmk_core/protocol/vusb/usbdrv/usbdrv.c

## Types of Changes

- [x] Core
- [ ] Bugfix
- [x] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/plaid/keymaps/default/config.h [hunk 1]
```diff
@@ -0,0 +1,19 @@
+/* Copyright 2019 REPLACE_WITH_YOUR_NAME
```
Comment (mechmerlin) [inline keyboards/plaid/keymaps/default/config.h]:
```suggestion
/* Copyright 2019 Takuya Urakawa (dm9records.com)
```

File: keyboards/plaid/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,222 @@
+/* Copyright 2019 REPLACE_WITH_YOUR_NAME
```
Comment (mechmerlin) [inline keyboards/plaid/keymaps/default/keymap.c]:
```suggestion
/* Copyright 2019 Takuya Urakawa (dm9records.com)
```

File: keyboards/plaid/keymaps/default/config.h (hunk 1 above)
Comment (hsgw) [reply keyboards/plaid/keymaps/default/config.h]: Thank you. I replaced.

File: keyboards/plaid/keymaps/default/keymap.c (hunk 2 above)
Comment (hsgw) [reply keyboards/plaid/keymaps/default/keymap.c]: Thank you. I replaced.

**Pull request merged**